helper-cli/src/main/kotlin/commands/packagecuration/CreateCommand.kt

```diff
@@ -20,20 +20,21 @@
 package org.ossreviewtoolkit.helper.commands.packagecuration
 
 import com.github.ajalt.clikt.core.CliktCommand
-import com.github.ajalt.clikt.core.UsageError
 import com.github.ajalt.clikt.parameters.options.convert
 import com.github.ajalt.clikt.parameters.options.option
 import com.github.ajalt.clikt.parameters.options.required
 import com.github.ajalt.clikt.parameters.types.file
 
 import java.io.IOException
+import java.lang.IllegalArgumentException
```

Comment:
Can you give a hint on the use case for the added logic?

Reply:
The curation files support multiple entries by file.
This enhances the `create` command to add new curations to this file.

the file is being created in `<type>/<namespace>/<name>.yml` and multiple versions of packages live in this file.

Reply:
Let's assume the file already exist and thus probably has >= 1 curations in it.
The use case is that a person wants to add a curation to that file. That user has to 

1. find the file location
2. open the file
3. insert a new curation

The rationale would be: the command helps with 1 / 2 since it prints the file location.
It helps with 3. by saving a copy and paste step. Instead of the user copying the curation
from directly above a template is already inserted. So was the intention of this commit to speed up 1,2,3?
Or maybe just 1 / 2 ?

Reply:
I wanted to cover all 3 mentioned use cases.

Additionally it covers use case 4 for me:
As a user that writes a curation, I don't need to know if a curation file already exists. The command behaves the same if a new file is created or if one already exists.

Reply:
Thanks! makes sense now.